graphstorm-processing/graphstorm_processing/graph_loaders/dist_heterogeneous_loader.py

```diff
@@ -703,7 +705,7 @@ def _extend_mapping_from_edges(
             map_schema = StructType(
                 [
                     StructField(join_col, StringType(), True),
-                    StructField(index_col, IntegerType(), True),
+                    StructField(index_col, LongType(), True),
```

Comment:
Why we want to change IntegerType to LongType?

Reply:
These are the mappings from string id to integer id. IntegerType is a signed 32bit integer, so only allows ~2B nodes to be mapped, whereas LongType is a 64bit int.

Reply:
Make sense.